packages/common/src/interfaces/column.interface.ts

```diff
@@ -108,6 +108,9 @@ export interface Column<T = any> {
   /** Default to false, which leads to exclude the column title from the Grid Menu. */
   excludeFromGridMenu?: boolean;
 
+  /** Defaults to false, which leads to exclude the field property, but including the fields property, from the query (typically a backend service query) */
```

Comment:
can you use backticks on the column options, when we use that it shows them as code in the editor as well
```ts
/** ... which leads to exclude the `field` property, but including the `fields` property
```
Could you also update the next option I created as well. I started using this approach not long ago, I didn't know VSCode also shows code this way :)  see example below, thought it's hard to see, the field has a color background

![image](https://github.com/ghiscoding/slickgrid-universal/assets/643976/4b4438df-2bed-429e-91c8-38e4352c0a81)